### PR TITLE
Include timestamp in EventEnvelope, #668

### PR DESCRIPTION
**Pull request by patriknw**
* timestamp extracted from timeUUID field
* requires Akka 2.5.29

Refs #668

To be forward ported to master.

### CODE REVIEW

**Comment by chbatey** [comment]
needs a format

**Comment by patriknw** [comment]
some travis problem, will rebuild later: unable to access 'https://github.com/akka/akka-persistence-cassandra.git/'

File: core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala [hunk 1]
```diff
@@ -424,6 +445,27 @@ class EventsByTagSpec extends AbstractEventsByTagSpec(EventsByTagSpec.config) {
       })
     }
 
+    "include timestamp in EventEnvelope" in {
```
Comment (renatocaval) [inline core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala]: that test name is already in use

Comment (renatocaval) [reply core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala]: more than that, we have it twice

Comment (patriknw) [reply core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala]: it's two different test sections, one for `currentEventsByTag` and another for `eventsByTag`

**Comment by chbatey** [comment]
@patriknw  there's an issue with the Scala upgrade, incompatible with silencer so that'll need to be upgraded too

File: core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala (hunk 1 above)
Comment (renatocaval) [reply core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala]: Ah, sorry. I haven't expanded the full context in GitHub.

**Comment by patriknw** [comment]
Attempt to fix the silencer dependency. Same as in akka/akka.

**Pull request merged**